gammapy/datasets/tests/test_io.py

```diff
@@ -124,16 +125,16 @@ def test_spectrum_datasets_to_io(tmp_path):
 @requires_data()
 def test_ogip_writer(tmp_path):
     dataset = SpectrumDatasetOnOff.read(
-        "$GAMMAPY_DATA/joint-crab/spectra/hess/pha_obs23523.fits"
+        "$GAMMAPY_DATA/joint-crab/spectra/hess/pha_obs23523.fits",
+        format="ogip",
     )
-    dataset.counts_off = None
+    dataset.counts_off.data = np.zeros(dataset.counts_off.data.shape)
```

Comment:
Why changing this specific test?

Reply:
see https://github.com/gammapy/gammapy/pull/5812#issuecomment-2807431211